src/v2/Apps/Home/Components/HomeFeaturedMarketNews.tsx

```diff
@@ -20,22 +20,31 @@ import {
 import { compact, take } from "lodash"
 import * as React from "react";
 import { createFragmentContainer, graphql } from "react-relay"
+import { StyledLink } from "v2/Components/Links/StyledLink";
 import { Masonry } from "v2/Components/Masonry"
 import { useSystemContext, useTracking } from "v2/System"
 import { SystemQueryRenderer } from "v2/System/Relay/SystemQueryRenderer"
 import { RouterLink } from "v2/System/Router/RouterLink"
 import { Media } from "v2/Utils/Responsive"
 import { HomeFeaturedMarketNewsQuery } from "v2/__generated__/HomeFeaturedMarketNewsQuery.graphql"
 import { HomeFeaturedMarketNews_articles } from "v2/__generated__/HomeFeaturedMarketNews_articles.graphql"
+import styled from "styled-components"
 
 const ARTICLE_COUNT = 6
 
 interface HomeFeaturedMarketNewsProps {
   articles: HomeFeaturedMarketNews_articles
 }
 
+const StyledTitle = styled(StyledLink)`
+  text-decoration: none;
+  &:hover {
+    color: ("blue100");
```

Comment:
To set something as our themed blue you need to use `themeGet`

```
color: ${themeGet("colors.blue100")};
```

Which is a function that is bound to our underlying theme and let's us access properties from it.